test/parallel/test-performance-gc.js

```diff
@@ -51,3 +51,13 @@ const kinds = [
   // Keep the event loop alive to witness the GC async callback happen.
   setImmediate(() => setImmediate(() => 0));
 }
+
+// GC should not keep the event loop alive
+{
+  let didCall = false;
+  process.on('beforeExit', () => {
+    assert(!didCall);
+    didCall = true;
```

Comment:
You could just use `common.mustCall()` on the `beforeExit` handler instead of tracking via `didCall`, right?

Reply:
Nope, that would create an infinite loop of always doing more GC and restarting `beforeExit`.